code/implementation/docker/compute/src/main/java/io/cattle/platform/docker/process/instance/DockerInstancePreCreate.java

```diff
@@ -77,6 +82,15 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
         return new HandlerResult(data).withShouldContinue(true);
     }
 
+    private Boolean removeLabel(Boolean isNativeContainer, String data_mode) {
+        if(!isNativeContainer) {
+            return true;
+        } else if (data_mode.equals("host") && data_mode.equals("container")) {
```

Comment:
I think we have host/container under constants somewhere, NetworkConstants? Hardcoding is undesirable.